### PR TITLE
Removing templates and wizards

### PR DESCRIPTION
**Pull request by singhsarab**
Removing the templates from this repo as these have been moved to http://github.com/dotnet/test-templates

Fixes : https://github.com/Microsoft/testfx/issues/598

### CODE REVIEW

File: scripts/UpdateTemplates.ps1 [hunk 1]
```diff
@@ -1,284 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (jayaranigarg) [inline scripts/UpdateTemplates.ps1:1]: Should we check-in this script in test-templates repo? We will have to create  nuget updations PRs there, this script can help us automate that! Though script will need some modifications as per the structure of the new repo.

File: scripts/build/TestFx.Settings.targets [hunk 2]
```diff
@@ -7,7 +7,6 @@
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props') and $(FrameworkIdentifier) != 'NetCore'" />
   <!-- Import props/targets with $(RepoRoot) since msbuild takes the relative path based on settings.targets and not with respect to the project. -->
   <Import Project="$(RepoRoot)packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props" Condition="Exists('$(RepoRoot)packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props')" />
-  <Import Project="$(RepoRoot)packages\microsoft.vssdk.buildtools.15.8.3247\build\Microsoft.VSSDK.BuildTools.props" Condition=" $(IsVsixProj) == 'true' and Exists('$(RepoRoot)packages\microsoft.vssdk.buildtools.15.8.3247\build\Microsoft.VSSDK.BuildTools.props')" />
```
Comment (jayaranigarg) [inline scripts/build/TestFx.Settings.targets:10]: TestFx.targets and TestFx.Loc.Targets will need some cleanup as well.

**Comment by jayaranigarg** [comment]
Removing ~180 files, super cool 🥇

File: scripts/UpdateTemplates.ps1 (hunk 1 above)
Comment (singhsarab) [reply scripts/UpdateTemplates.ps1:1]: Not required. VS code works better.

File: scripts/build/TestFx.Settings.targets (hunk 2 above)
Comment (singhsarab) [reply scripts/build/TestFx.Settings.targets:10]: Thanks. I actually missed pushing these.

**Pull request merged**